janitor-test/main.tf

```diff
@@ -0,0 +1,12 @@
+/**
+ * # janitor-test
```

Comment:
there's no terraform in this file

Reply:
But this is a terraform module. Do you want me add some comments in this file? Or remove something? 
I just run terraform-doc, nothing changed

Reply:
Sorry, what do you mean? Is main.tf autogenerated by terraform-doc, even though there's no terraform code in it? Or do we need this empty file to make terraform-doc happy?

Reply:
Wrong comments. This is not generated by terraform-doc. @gmalkov to see if I can delete the main file

Reply:
See the terraform-docs docs:
https://github.com/terraform-docs/terraform-docs/blob/master/docs/USER_GUIDE.md#generate-module-header

Instead of having different files for each module to point it to, I prefer to just always have a main.tf file that has the doc info.